android/apollo/src/com/andrew/apollo/MusicPlaybackService.java

```diff
@@ -455,16 +434,14 @@
 
     private int mMediaMountedCount = 0;
 
-    private int mShuffleMode = SHUFFLE_NONE;
+    private boolean mShuffleMode = false;
```

Comment:
rename to `mShuffleEnabled` or `mShuffleOn`, no need for "modes", it's a switch.